### PR TITLE
Fixing windows integration tests.  Tests are being run properly now f…

### PR DESCRIPTION
**Pull request by aaron-prindle**
…or hyperv and things are being properly uploaded to gcs/github.  There are compatibilty issues though with the actual tests that I will attempt to fix in this PR as well

### CODE REVIEW

**Comment by aaron-prindle** [comment]
- [ ] Add windows back into job pipeline when complete

**Comment by dlorenc** [comment]
Could you rebase this on @r2d4's latest timeout change?

**Comment by dlorenc** [comment]
I don't see the windows run reporting results, should it be yet?

**Comment by aaron-prindle** [comment]
PTAL, Windows tests are working now.

**Comment by aaron-prindle** [comment]
I believe this is XHyve flake

File: test/integration/addons_test.go [hunk 1]
```diff
@@ -75,8 +75,7 @@ func TestDashboard(t *testing.T) {
 		BinaryPath: *binaryPath,
 		Args:       *args,
 		T:          t}
-	minikubeRunner.Start()
-	minikubeRunner.CheckStatus("Running")
+	minikubeRunner.EnsureRunning()
```
Comment (dlorenc) [inline test/integration/addons_test.go]: Could you open an issue for the start/start issue?

**Pull request merged**